src/lib.rs

```diff
@@ -271,6 +269,19 @@ pub async fn run(command: Subcommands) -> Result<(), SoldeerError> {
             }
         }
         Subcommands::Push(push) => {
+            let path = push
+                .path
+                .unwrap_or(get_current_working_dir().to_str().unwrap().to_string());
+            let path_buf = PathBuf::from(&path);
+
+            // Check for sensitive files or directories
+            if utils::check_for_sensitive_files_or_directories_recursive(&path_buf) {
```

Comment:
```suggestion
            if check_for_sensitive_files_or_directories_recursive(&path_buf) {
```

Sorry i just meant like this.

Reply:
also the name is a bit tbh, can we work on something smaller?

Reply:
Oh ok I get you now, also, yeah the name is way too descriptive XD